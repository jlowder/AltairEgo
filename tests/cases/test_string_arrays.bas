10 REM Test string arrays
20 DIM A$(5)
30 DIM Q(5)
40 LET L$="HELLO"
50 FOR I=1 TO 5
60 LET Q(I)=I
70 A$(I)=MID$(L$,Q(I),1)
80 PRINT "A$(";I;")=";A$(I)
90 NEXT I
100 REM Test reading back
110 FOR I=1 TO 5
120 PRINT A$(I);
130 NEXT I
140 PRINT
150 END
