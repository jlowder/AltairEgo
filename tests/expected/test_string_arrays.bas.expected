Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
A$( 1 )=H
A$( 2 )=E
A$( 3 )=L
A$( 4 )=L
A$( 5 )=O
HELLO
OK
